### ISSUE TITLE
Multiple metrics in test, score and gridsearch

### ISSUE THREAD

**Issue by thomasfrederikhoeck** [andersbogsnes/ml_tooling#126]
#### Problem description

It would be convinitent to have the option to supply more than one metric as in scikit-learn. One could be used for best model and the rest would just be displayed.

---------
- [ ] I have updated ml-tooling to the newest version `pip install ml-tooling --upgrade`
- [ ] I have checked that this issue does not already exist

**Comment by andersbogsnes** [comment]
Need to decide how to handle results when using multiple metrics.

Current implementation returns a list-like of all results, sorting by `.score` attribute. 

Questions to be answered:
- How do we compare scores across metrics? 
- Should `.score` be instead a dict of  scores for each metric?
- Should we return one result for each of the metrics?
- How to indicate which metric to use for best model?

I'm currently leaning towards making one result per metric and then implementing sorting based on both metric name and score and just assuming the first metric is the most important, but I'm open to input!

For reference - sklearn's implementation: 
https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/model_selection/_validation.py

**Comment by thomasfrederikhoeck** [comment]
For gridsearch I like scikit learns approach with a DataFrame (not actually DataFrame both formatted to become a DF in a oneliner) where rows are each point in the parameter grid and then columns with each metric, where the one used for finding the best model is specified with the `refit` option. This could also  be logged easily .

**Comment by andersbogsnes** [comment]
closed by #290
